docs/administration-guide/databases/mongodb.md

```diff
@@ -1,27 +1,36 @@
 # Working with MongoDB in Metabase
 
+This article covers:
 
-This guide covers:
-
+ - Using a connection string.
  - General connectivity concerns.
  - Connecting to a MongoDB Atlas cluster.
 
+## Using a connection string
+
+If you'd prefer to connect to MongoDB using a [connection string](https://docs.mongodb.com/manual/reference/connection-string/),
+click on **Paste a connection string**. The Metabase UI will update with a field to paste your connection string, as well as present the following options:
+
+- **Use an SSH tunnel for database connections.** Some database installations can only be accessed by connecting through an SSH bastion host. This option also provides an extra layer of security when a VPN is not available. Enabling this is usually slower than a direct connection.
+- **Automatically run queries when doing simple filtering and summarizing.** When this is on, Metabase will automatically run queries when users do simple explorations with the Summarize and Filter buttons when viewing a table or chart. You can turn this off if querying this database is slow. This setting doesn’t affect drill-throughs or SQL queries. 
+- **This is a large database, so let me choose when Metabase syncs and scans.** By default, Metabase does a lightweight hourly sync and an intensive daily scan of field values. If you have a large database, we recommend turning this on and reviewing when and how often the field value scans happen.
+
 ## General connectivity concerns
 
- - Version `0.32.9` introduced an option to connect using `DNS SRV`, which is the recommended method for newer Atlas clusters.
+ - **Connect using `DNS SRV`**, which is the recommended method for newer Atlas clusters.
 
- - Have you checked your cluster host whitelist?  When testing a connection but seeing failure, have you tried setting the IP whitelist to `0.0.0.0/0`?  This allows connections from any IP addresses. If you know the IP address(es) or CIDR block of clients, use that instead.
+ - **Have you checked your cluster host whitelist?** When testing a connection but seeing failure, have you tried setting the IP whitelist to `0.0.0.0/0`?  Whitelisting this address allows connections from any IP addresses. If you know the IP address(es) or CIDR block of clients, use that instead.
 
- - Are you using self-signed certificates?  While Metabase doesn't support providing a self-signed cert through the Metabase UI, you can still use the command line to configure this.  Just copy the existing store and add the self-signed cert to it like this:
+ - **Are you using self-signed certificates?** While Metabase doesn't support providing a self-signed cert through the Metabase UI, you can still use the command line to configure this. Just copy the existing store and add the self-signed cert:
```

Comment:
This section needs to be changed as a result of #12564 - we now support providing the mongo cert in the UI (but only when using the individual fields view, not the connection string view)

Reply:
Thanks for the notes, @robdaemon. I rearranged the doc and added some navigation. Called out the fact that you can set up SSL via the default method and NOT via a connection string. I also carved out adding a cert via the command line as its own section.